mcs/class/System/System.Net.Sockets/Socket.cs

```diff
@@ -1118,8 +1118,14 @@ static SafeSocketHandle Accept_internal (SafeSocketHandle safeHandle, out int er
 
 #region Bind
 
+#if FEATURE_NO_BSD_SOCKETS
+		[Obsolete  ("System.Net.Sockets.Socket:Bind is not supported on this platform.")]
+#endif
```

Comment:
You mean I should remove the Obsolete or edit the message to match other PNSE's (`System.Net.Sockets.Socket:Bind is not supported on the current platform.`)?